src/Client.php

```diff
@@ -17,13 +17,10 @@
 
 namespace MongoDB;
 
-use Composer\InstalledVersions;
 use Iterator;
-use MongoDB\BSON\Document;
-use MongoDB\BSON\PackedArray;
-use MongoDB\Builder\BuilderEncoder;
 use MongoDB\Builder\Pipeline;
-use MongoDB\Codec\Encoder;
+use MongoDB\Builder\Stage;
```

Comment:
I don't think this class is used in the `Client` class. PHPCS may be confused because you declare the `stage` type.